app/src/main/java/com/gdsctsec/smartt/ui/main/weekday/WeekdayActivity.kt

```diff
@@ -24,9 +24,9 @@ class WeekdayActivity : AppCompatActivity() {
         setContentView(R.layout.activity_weekday)
 
         val imageView: ImageView = findViewById(R.id.empty_list_image_view)
-        val lecturesRecyclerView : RecyclerView = findViewById(R.id.recyclerView)
+        val lecturesRecyclerView: RecyclerView = findViewById(R.id.lecture_recycler_view)
         val lecNumber: TextView = findViewById(R.id.lecture_number_text_view)
-        var timeList: FloatingActionButton = findViewById(R.id.lecture_add_floating_action_button)
+        val timeList: FloatingActionButton = findViewById(R.id.lecture_add_floating_action_button)
 
         //recycler View Adapter
         val timeListRecyclerView: List<String> = listOf(
```

Comment:
?

Reply:
changed it to val cause dont need to change it anywhere